mobile/mobile.go

```diff
@@ -76,8 +76,9 @@ type MigrateConfig struct {
 
 // RunConfig is used to define run options for a mobile node
 type RunConfig struct {
-	RepoPath string
-	Debug    bool
+	RepoPath          string
+	Debug             bool
+	CafeOutboxHandler core.CafeOutboxHandler
```

Comment:
Ok so `CafeOutboxHandler` just specifies that the object needs to implement `Flush()`... Is the idea that `Flush` will be called whenever new `CafeRequest`s are added to the queue?

Reply:
Yep, exactly.